### PR TITLE
Send SNS messages from chef run

### PR DESCRIPTION
**Pull request by fh**

### CODE REVIEW

**Comment by baccenfutter** [comment]
@fh: virtually re-assigning to you. :)

File: easybib/libraries/sns.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+require 'json'
+
+module EasyBib
+  module SNS
+    def self.sns_notify(node, body)
+      require 'aws-sdk'
+
+      if node.nil?
+        Chef::Log.error 'Missing argument: node (AWS instance name)!'
```
Comment (fh) [inline easybib/libraries/sns.rb]: @baccenfutter This still will continue and therefore lead to a broken deploy. better return here, too :)

Comment (fh) [inline easybib/libraries/sns.rb]: Also maybe check for both `node['easybib']['sns']['credentials']` and `node['easybib']['sns']['topic']`not being nil. you can use ruby-fu here: `if node.fetch('easybib', {}).fetch('sns', {})['credentials'].nil?`

**Comment by fh** [comment]
@baccenfutter Please remove the WIP from the subject if this should be merged :)

**Pull request merged**